prow/config/tide.go

```diff
@@ -146,6 +151,10 @@ type Tide struct {
 	//  0 => unlimited batch size
 	// -1 => batch merging disabled :(
 	BatchSizeLimitMap map[string]int `json:"batch_size_limit,omitempty"`
+
+	// Priority is an ordered list of labels that would be prioritized before other PRs
+	// PRs should match all labels contained in a list to be prioritized
```

Comment:
nit: This doesn't seem quite accurate.
```suggestion
	// Priority is an ordered list of sets of labels that would be prioritized before other PRs
	// PRs should match all labels contained in a set to be prioritized
```